docs/geedocs/docsrc/answer/geeServerConfigAndSecurity/postgreSqlAuthentication.rst

```diff
@@ -113,7 +113,7 @@ PostgreSQL authentication
          ``psql --port=5433 --username=geeuser geauthdb < /usr/share/postgresql/8.4/contrib/pgcrypto.sql``
 
          Note: You can substitute a different method for this step. The `Password Formats
-         <http://www.google.com/url?q=http%3A%2F%2Fhttpd.apache.org%2Fdocs%2F2.2%2Fmisc%2Fpassword_encryptions.html&sa=D&sntz=1&usg=AFrqEzdBJJpsOLV3eL6UCAatZv_IhxEZdg>`__
+         <http://www.google.com/url?q=http%3A%2F%2Fhttpd.apache.org%2Fdocs%2F2.4%2Fmisc%2Fpassword_encryptions.html&sa=D&sntz=1&usg=AFrqEzdBJJpsOLV3eL6UCAatZv_IhxEZdg>`__
```

Comment:
Direct url ? https://httpd.apache.org/docs/2.4/misc/password_encryptions.html#:~:text=Apache%20recognizes%20one%20format%20for,the%20AuthName%20directive%20in%20httpd.